### PR TITLE
Cleans up old sign up user type experiment

### PR DESCRIPTION
**Pull request by hannahbergam**
The sign up flow piloted a new user type button (see image below) a couple years ago. The DCDO flag was flipped to 'true' on production, but now we have an inconsistent user experience on test and locally (which are still defaulted to the old dropdown selection). Because the work I'm doing with School Association is safer with a comprehensive UI test, the test I wrote requires drone to assume the 'new' code that matches production (ie clicking on the teacher button instead of using the old dropdown). So, I'm cleaning it up!

The DCDO flag appears to be working through [the logic within the experiments file](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/util/experiments.js#L167) that checks for a dcdo flag by the same name. I could not find any direct/other references to that flag. 

Because this is a high traffic area, it would be great if reviewers could run the same search locally to ensure I'm not missing any surprise use cases!

Able to confirm that my local experience is the cards with this change:
Screenshot 2024-04-09 at 12 50 50 PM

## Warning!!

The [AP CSP Create Performance Task](https://apcentral.collegeboard.org/courses/ap-computer-science-principles/exam) is in progress. The most critical dates are from April 3 - April 30, 2024. Please consider any risk introduced by this PR that could affect our students taking AP CSP. Code.org students taking AP CSP primarily use App Lab for their Create Task, however a small percent use Game Lab. Carefully consider whether your change has any risk of alterering, changing, or breaking anything in these two labs. Even small changes, such as a different button color, are considered significant during this time period. Reach out to the Student Learning team or Curriculum team for more details.

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1703

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js [hunk 1]
```diff
@@ -59,23 +54,13 @@ $(document).ready(() => {
...
-      } else {
-        // Otherwise (also the default), keep original dropdown
-        document.getElementById('select-user-type-variant').style.cssText =
-          'display:none;';
-        document.getElementById('select-user-type-original').style.cssText =
-          'display:flex;';
-        document.getElementById('signup-select-user-type-label').style.cssText =
-          'width:220px;';
-      }
+      // If in variant, toggle large buttons
```
Comment (TurnerRiley) [inline apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js]: Nit: I think this comment can be deleted too!

**Review by TurnerRiley — APPROVED** [review]
Works for me locally too!

File: apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js (hunk 1 above)
Comment (hannahbergam) [reply apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js]: Ahh good catch! Updated!

**Review by bethanyaconnor — APPROVED** [review]
Yay for cleaning up old code!!

**Pull request merged**